tests/define_aggregate_test_project.py

```diff
@@ -13,6 +13,8 @@ def statepoint_i_even_odd_aggregator(jobs):
 
 def set_all_job_docs(jobs, key, value):
     for job in jobs:
+        if key in job.document and job.document[key] == value:
+            raise RuntimeError(f"Key {key} is already set to {value}!")
```

Comment:
```suggestion
        if job.document.get(key) == value:
            raise RuntimeError(f"Key {key} is already set to {value}!")
```

Reply:
This refactor isn't quite the same, because it would not work if `value is None`. That was why I wrote it like this in the first place. However, since we aren't using `None` as a value and this is a bit simpler, I will apply this change.